app/src/main/java/org/metabrainz/mobile/util/Utils.kt

```diff
@@ -31,6 +45,53 @@ object Utils {
         return Intent(Intent.ACTION_VIEW, Uri.parse(url))
     }
 
+    fun sendToPicard(context: Context, releaseMBID: String){
+        val ipAddress = UserPreferences.preferenceIpAddress
+        if(ipAddress==null){
+            Toast.makeText(context,"Add your IP Address in the settings, matched according to your Picard network", Toast.LENGTH_LONG).show()
+            return
+        }
+        //To allow http requests specially
+        val policy = StrictMode.ThreadPolicy.Builder().permitAll().build()
+        StrictMode.setThreadPolicy(policy)
+
+        val url = String.format(
+            App.PICARD_OPENALBUM_URL, ipAddress,
+            UserPreferences.preferencePicardPort, uriEncode(releaseMBID)
+        )
+
+        CoroutineScope(context = Dispatchers.Default).launch {
+            var connection: HttpURLConnection? = null
+            try {
+                val u = URL(url)
+                connection = u.openConnection() as HttpURLConnection?
```

Comment:
I think its better to do the request using `OkHttp`.

Reply:
I agree using `OkHttp` makes more sense. I have made the relevant changes!